packages/playground/php-cors-proxy/cors-proxy.php

```diff
@@ -255,7 +262,9 @@ function(
 // Close cURL session
 curl_close($ch);
 
-// Only send chunked transfer encoding footer if we're using chunked encoding
-if ($is_chunked_response) {
+// Only send chunked transfer encoding footer if we're using chunked encoding.
+// We need to manually send the footer when running in the PHP built-in server
+// because, unlike apache or nginx, it won't handle that for us.
+if ($is_chunked_response && php_sapi_name() === 'cli-server') {
```

Comment:
I meant to add this under the code review response...

nit: The terms of this condition are used in multiple places. Maybe it would be good to capture it in a function like `should_send_as_chunked_response()`.